automation/qbft/scenarios/sync_failover.go

```diff
@@ -146,7 +146,8 @@ loop:
 	}
 	r.logger.Info("node #4 synced", zap.Int64("highest decided", int64(nextSeq)-1))
 
-	decides, err := ctx.Stores[3].GetDecided(message.ToMessageID(msgs[1].Message.Identifier), 0, nextSeq)
+	messageID := message.ToMessageID(msgs[1].Message.Identifier)
```

Comment:
@nkryuchkov 
we need to change the `msgs[1].Message` to use spec message struct

Reply:
@nivBlox Do you mean to change `*specqbft.SignedMessage` to `*specqbft.Message`?

Reply:
@nkryuchkov yes